packages/libs/error-stack/src/hook/mod.rs

```diff
@@ -181,4 +183,50 @@ impl Report<()> {
 
         closure(&hook)
     }
+
+    #[cfg(all(any(feature = "std", feature = "hooks"), feature = "serde"))]
+    pub fn install_serde_hook<T: serde::Serialize + Send + Sync + 'static>() {
+        crate::serde::install_builtin_hooks();
```

Comment:
Same as above, could we also fully qualify the debug hooks or rename the functions?

Reply:
fixed in [83edbf2](https://github.com/hashintel/hash/pull/1558/commits/83edbf2a5191c8c8b38615601044655aea37277f)